docs/debugging/index.md

```diff
@@ -34,11 +36,13 @@ Table of contents:
 # 🤯 0.9092974662780762 🤯
   ```
 
-Click [here](print_breakpoint) to learn more!
+[Read more](print_breakpoint).
```

Comment:
I'd consider using a sphinx `:ref:` for this link, rather than linking via the filename.

Reply:
That's useful in general. In this case I want the anchor text to say "read more," similar to what it did before this change.